plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemSignIsWaxed.java

```diff
@@ -0,0 +1,55 @@
+package com.denizenscript.denizen.objects.properties.item;
+
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.Tag;
+import org.bukkit.block.Sign;
+import org.bukkit.inventory.meta.BlockStateMeta;
+
+public class ItemSignIsWaxed extends ItemProperty<ElementTag> {
+
+    // <--[property]
+    // @object ItemTag
+    // @name is_waxed
+    // @input ElementTag(Boolean)
+    // @description
+    // Controls whether a sign item is waxed (cannot be edited once placed).
+    // @tag
+    // Will return null if the item doesn't have a waxed state set.
+    // -->
+
+    public static boolean describes(ItemTag item) {
+        return Tag.ALL_SIGNS.isTagged(item.getBukkitMaterial());
+    }
+
+    @Override
+    public ElementTag getPropertyValue() {
+        BlockStateMeta stateMeta = as(BlockStateMeta.class);
+        if (!stateMeta.hasBlockState()) {
+            return null;
```

Comment:
It makes sense for the property value to be unset in this case, but the tag would probably be better as false, as when you place the sign it will either be waxed or not, and lacking a block state I believe reliably indicates not waxed.

Reply:
Fair enough, done